addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,796 @@
+        <t t-call="website_event.index_sidebar"/>
     </xpath>
 </template>
 
-<template id="event_right_photos" inherit_id="website_event.event_right_column" active="False" customize_show="True" name="Photos">
-    <xpath expr="//div[@id='right_column']" position="inside">
-        <figure class="figure o_figure_relative_layout w-100">
-            <img class="figure-img img-fluid" src="/website_event/static/src/img/openerp_enterprise_of_the_year.png" alt=""/>
-            <figcaption class="figure-caption text-center"><a href="/event">Photos of Past Events</a></figcaption>
-        </figure>
-        <figure class="figure o_figure_relative_layout w-100">
-            <img class="figure-img img-fluid" src="/website_event/static/src/img/training.jpg" alt=""/>
-            <figcaption class="figure-caption text-center"><a href="/event">Our Trainings</a></figcaption>
-        </figure>
-    </xpath>
+<!-- Index Topbar -->
+<template id="index_topbar" name="Topbar">
+    <nav class="navbar navbar-light border-top shadow-sm d-print-none">
+        <div class="container">
+            <span class="navbar-brand h4 my-0 mr-4">Events</span>
+            <ul id="o_we_index_topbar_filters" class="nav mr-n2"/>
+            <t t-call="website_event.search_box">
+                <t t-set="_classes" t-value="'form-inline ml-auto pl-lg-3'"/>
+            </t>
+        </div>
+    </nav>
 </template>
 
-<template id="event_right_quotes" inherit_id="website_event.event_right_column" active="False" customize_show="True" name="Quotes">
-    <xpath expr="//div[@id='right_column']" position="inside">
-        <div class="card mb-3 bg-secondary">
-            <div class="card-body">
-                <blockquote class="m-0">
-                    <p>
-                        <i>Write here a quote from one of your attendees.
-                        It gives confidence in your
-                        events.</i>
-                    </p>
-                    <small>Author</small>
-                </blockquote>
+<!-- Filter - Date -->
+<template id="event_time" inherit_id="website_event.index_topbar" customize_show="True" name="Filter by Date">
+    <xpath expr="//ul[@id='o_we_index_topbar_filters']" position="inside">
+        <li class="nav-item dropdown mr-2 my-1 my-lg-0">
+            <a href="#" role="button" class="dropdown-toggle" data-toggle="dropdown">
+                <i class="fa fa-calendar"/>
+                <t t-if="current_date" t-esc="current_date"/>
+                <t t-else="">Next Events</t>
+            </a>
+            <div class="dropdown-menu">
+                <t t-foreach="dates" t-as="date">
+                    <t t-if="date[3] or (date[0] in ('old','all'))">
+                        <a t-attf-href="/event?{{ keep_query('country', 'type', date=date[0]) }}" t-attf-class="dropdown-item d-flex align-items-center justify-content-between #{searches.get('date') == date[0] and ' active' or ''}">
```

Comment:
`'active'` instead of `' active'` (always. We should never do something like that: `my_classname#{A and ' another_class'}`, always `my_classname #{A and 'another_class'}`. I used to do the mistake but it is not good for technical reasons).